annotation-processing/src/main/java/com/exactpro/epfast/annotation/processing/FastTypeElement.java

```diff
@@ -1,17 +1,33 @@
 package com.exactpro.epfast.annotation.processing;
 
+import com.exactpro.epfast.annotations.FastField;
 import com.exactpro.epfast.annotations.FastType;
 
 import javax.lang.model.element.Element;
+import java.util.ArrayList;
+import java.util.List;
 
 public class FastTypeElement {
     private String fastTypeName;
 
     private Element fastTypeElement;
 
+    private List<FastFieldElement> fastFields;
+
     public FastTypeElement(Element fastTypeElement) {
         this.fastTypeName = getNameFromElement(fastTypeElement);
         this.fastTypeElement = fastTypeElement;
+        this.fastFields = processFastFields(fastTypeElement);
+    }
+
+    private List<FastFieldElement> processFastFields(Element fastTypeElement) {
+        ArrayList<FastFieldElement> fastFieldElements = new ArrayList<>();
+        for (Element element : fastTypeElement.getEnclosedElements()) {
```

Comment:
It looks you've broken fields calculation into two parts. It's bad idea. The code should be in one place. And it appears that it shouldn't be in this class.